modern/src/settings/components/CollectionFab.js

```diff
@@ -11,7 +11,7 @@ const useStyles = makeStyles((theme) => ({
     bottom: theme.spacing(2),
     right: theme.spacing(2),
     [theme.breakpoints.down('md')]: {
-      bottom: theme.dimensions.bottomBarHeight + theme.spacing(2),
+      bottom: parseInt(theme.dimensions.bottomBarHeight, 10) + parseInt(theme.spacing(2), 10),
```

Comment:
I commented on the other PR. You should use `calc` option here instead of parsing values.